### ISSUE TITLE
Niche Market

### ISSUE THREAD

**Issue by wsjackson14** [oats-group3/General-Work#12]
How do you plan on getting into the online sales world and compete with Amazon or Walmart.com? I understand you only choose the highest quality items and that's supposed to set you apart, but what makes you think wealthy people would trust your opinions without your brand being recognizable? Also I would still try and market to the middle and lower classes as well, as it gives you a much larger customer base and allows you to grow your business. Just because you choose high quality items doesn't mean that they have to be super expensive.

**Comment by ChristinaNguyen9** [comment]
I agree with you on that part, there are markets out there that sell high quality products but are still reasonably priced that all classes are able to purchase them.
